src/main/resources/i18n/nls.properties

```diff
@@ -3,6 +3,7 @@ core=Core
 lwjgl2=Desktop (Legacy)
 lwjgl3=Desktop
 ios=iOS
+ios-moe=iOS Multi-Os Engine
```

Comment:
Pretty sure this should be OS (Operating System), but it isn't a big deal, I'll fix it after I merge.

Reply:
Ahh yes, my bad! Thanks!